tests/test_rf_base.py

```diff
@@ -231,6 +231,32 @@ def _forward_step(*, model: _Net, extern_data: TensorDict):
     run_model(extern_data, lambda *, epoch, step: _Net(), _forward_step)
 
 
+def test_linear():
```

Comment:
If you only want to test gradient checkpointing, `test_rf_gradient.py` is a better place for the test.